services/speech_to_text/v1.js

```diff
@@ -85,17 +91,18 @@ module.exports = function (RED) {
 
       var s2t = function (audio, format, cb) {
         if (format !== 'wav' && format !== 'flac' && format !== 'ogg') {
-          var message = 'Audio format (' + format + ') not supported, must be encoded as WAV, FLAC or OGG.';
-          node.error(message, msg)
+          var message_err_format = 'Audio format (' + format + ') not supported, must be encoded as WAV, FLAC or OGG.';
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'message_err_format' is not in camel case.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2391621463/issues/source?bid=3225115&fileBranchId=3229003#l95)